autoload/lightline/colorscheme/ayu_dark.vim

```diff
@@ -0,0 +1,39 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/ayu_dark.vim
+" Author: danielpeng2
+" License: MIT License
+" Last Change: 2020/04/15 20:35:50.
+" =============================================================================
+let s:base0 = [ '#e6e1cf', 244 ]
```

Comment:
The non-GUI colors are the copy of wombat colorscheme, right?

Reply:
Yes, the same non-GUI colors are also used for ayu_light and ayu_mirage. I can update them if you'd like.

Reply:
Hmm, okay, I have to fix all of them...